core/src/display_object/edit_text.rs

```diff
@@ -1785,6 +1785,18 @@ impl<'gc> EditText<'gc> {
         }
     }
 
+    fn on_scroller(&self, activation: &mut Avm1Activation<'_, 'gc>) {
+        if let Avm1Value::Object(object) = self.object() {
+            let _ = object.call_method(
+                "broadcastMessage".into(),
+                &["onScroller".into(), object.into()],
+                activation,
+                ExecutionReason::Special,
+            );
+        }
+        //TODO: Implement this for Avm2
```

Comment:
Is this just the mouse wheel event?

Reply:
It is indeed for the mouse scroll event. Your PR does the AVM2 side of this, right?

Reply:
> Is this just the mouse wheel event? 

AFAIK no, in AS3 a textfield has a separate `scroll` event.

That said, I don't think it's blocking this PR?
Though what probably should be investigated is the interaction of these two PRs, as in: whether capturing this event and `.preventDefault()`ing it is supposed to disable scrolling textfields.

Reply:
`MouseEvent.WHEEL` and `Event.SCROLL` cannot be `preventDefault`ed, so this should be fine.